bench/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "sanctuary-type-classes-benchmarks",
+  "version": "1.0.0",
```

Comment:
If this is not going to be released as a package, I think `"0.0.0"` would be more appropriate.